docker/wifi-connect/requirements.txt

```diff
@@ -0,0 +1 @@
+pyzmq==24.0.1
```

Comment:
Change the version if necessary.

Reply:
I've changed my base image to use Python 3.7 instead of 3.11, and using this version of `pyzmq` still works. Will leave the version as it is.